pkgs/misc/cups/drivers/cnijfilter_2_80/default.nix

```diff
@@ -100,6 +100,10 @@ stdenv.mkDerivation {
      them, it undoes the --set-rpath.  this prevents that. */
   dontPatchELF = true;
 
+  # fortify hardening makes the filter crash
+  # https://github.com/NixOS/nixpkgs/issues/276125
+  hardeningDisable = [ "fortify" ];
```

Comment:
Apparently we can get away with just disabling `fortify3`

Reply:
cool, I did not know about this one.